src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
+$text-color: #060b35;
+$button-color: #00acdc;
+$card-code-color: #616070;
+
+* {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: auto;
+
+  &__image {
+    display: flex;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    &_info {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      text-align: left;
+      justify-content: center;
+      margin-bottom: 4px;
+      color: $text-color;
+    }
+
+    &_code {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: left;
+      color: $card-code-color;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    .stars {
+      display: flex;
+
+      &__star {
+        width: 16px;
+        height: 16px;
+        background-repeat: no-repeat;
+        margin-right: 4px;
+        background-position: center;
+      }
+
+      &__star:nth-child(-n + 4) {
+        background-image: url(/src/images/star-active.svg);
+      }
+
+      &__star:nth-child(5) {
+        background-image: url(/src/images/star.svg);
+      }
+    }
+    .review {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: right;
+      color: $text-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    .card__text1 {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: $card-code-color;
+    }
+
+    .card__text2 {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: $text-color;
+    }
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    background-color: $button-color;
+    color: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
done